processmanager/ebpf/ebpf_test.go

```diff
@@ -38,6 +38,6 @@ func TestMapID(t *testing.T) {
 		})
```

Comment:
Suggestion: Add another testcase above:
```
		(1 << support.StackDeltaBucketLargest) - 1: support.StackDeltaBucketLargest,
```

Reply:
To clarify, it covers the corner case of the highest acceptable value, which was hard-coded previously.